CHANGES.md

```diff
@@ -6,6 +6,8 @@ Version 0.2.0
 
 To be released.
 
+ -  Change type of Peer.`Urls` to `IPEndPoint`.
```

Comment:
Please write the sentence in past tense and I think `Peer.Urls` should be quoted altogether.

> The type of `Peer.Urls` property is changed from `Uri` to `IPEndPoint`.